test/test_constraints.jl

```diff
@@ -261,6 +261,63 @@ end
         @test a ≈ aa ≈ al ≈ a_rhs1 ≈ a_rhs2
     end
 
+    # Test nonlinear solution procedure (on linear problem) with affine constraints 
+    # using standard assembly (i.e. not local condensation)
+    @testset "nonlinear" begin
+        params = (k=1.0, f=1.0, a=1.0, b=0.2, tol=1e-10, maxiter=2)
+        grid = generate_grid(Line, (2,)); addfaceset!(grid, "center", x->x[1]≈0.0)
+        dh = DofHandler(grid); push!(dh, :u, 1); close!(dh)
```

Comment:
Merged https://github.com/Ferrite-FEM/Ferrite.jl/pull/578 so:
```suggestion
        dh = DofHandler(grid); add!(dh, :u, 1); close!(dh)
```
(Maybe you need to rebase on master to get CI working after that change)